### PR TITLE
Added new feature to give feedback for community website 

### PR DESCRIPTION
**Pull request by Palaksharma23**
# Pull Request Template

## Description

I have added a floating icon for taking feedback from users, clicking on the feedback icon will lead users to a feedback form whose value you can check being updated in this API. 

https://sheet.best/api/sheets/3b6fc57e-5cd4-4fbf-b9de-4a58a8d93968

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [✅ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## Additional Context (Please include any Screenshots/gifs if relevant)

...

https://user-images.githubusercontent.com/87171452/165440969-6dccca65-20c9-4fb8-a802-9ad2430e0645.mp4

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

@HarshCasper may you please review this pull request :)

### CODE REVIEW

**Review by padmajabhol — CHANGES_REQUESTED** [review]

**Comment by Palaksharma23** [comment]
thank you @padmajabhol i will do the required changes :)

File: website/src/theme/Footer/index.js [hunk 1]
```diff
@@ -47,16 +53,114 @@ const FooterLogo = ({ sources, alt }) => (
...
+  const modalRef = useRef();
   const { copyright, links = [], logo = {} } = footer || {};
+  var feedback = ' ';
   const sources = {
     light: useBaseUrl(logo.src),
     dark: useBaseUrl(logo.srcDark || logo.src),
   };
 
+  const changeinput = (e) => {
+    console.log(e.target.value);
```
Comment (padmajabhol) [inline website/src/theme/Footer/index.js:65]: Do we need this?

File: website/src/theme/Footer/index.js [hunk 2]
```diff
@@ -47,16 +53,114 @@ const FooterLogo = ({ sources, alt }) => (
...
+    feedback = e.target.value;
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(feedback);
```
Comment (padmajabhol) [inline website/src/theme/Footer/index.js:71]: I don't think we need console.log to go into production.

**Review by padmajabhol — COMMENTED** [review]
Also is there a way you can improve the feedback button's UI? Sorry for the last review, was caught up.

**Comment by Palaksharma23** [comment]
sure @padmajabhol :) i will do the required changes

**Pull request closed without merging**